content/03.use-cases-a.md

```diff
@@ -5,12 +5,12 @@
 Many researchers often start their use of GitHub to backup their working code and data to a remote server (Just push and pull, see [Box 1](#definitions), from their own repo). 
 The GitHub repository serves as an additional backup and also allows the user to synchronize files and work on any computer with internet access. 
 Because GitHub is built on git, version control features are also available, allowing users to re-visit or even restore previous versions of the repository.
+
```

Comment:
I agree with spacing here, but feel that the comma needs to be added back in. We are saying that because GitHub is built on Git, version control is part of what you get.